chaincode/marbles_transfer/marbles_transfer.go

```diff
@@ -198,11 +199,44 @@ func agreeToPrice(ctx contractapi.TransactionContextInterface, marbleID string,
 	return nil
 }
 
-// TODO implement function to verify marble properties
-// For example, Org1 may tell Org2 about the properties and salt.
-// Org2 would want to verify the properties before agreeing to buy.
-// Org2 would call a verify function on his peer.
-// The properties and salt would passed in, get hashed in the chaincode, and compared with the on-chain hash of the marble properties (queried via GetPrivateDataHash).
+// VerifyAssetProperties implement function to verify marble properties
```

Comment:
Since this is a learning sample, a more descriptive comment would be good, explaining that the hash is verified.

Also, for each new function like this, add a call in the readme showing how it would be used.

Reply:
📚